packages/server/lib/plugins/child/run_require_async_child.js

```diff
@@ -73,11 +73,11 @@ function run (ipc, file, projectRoot) {
 
     if (devServer && typeof devServer === 'object') {
       if (devServer.bundler === 'webpack') {
-        return { devServer: require('@cypress/webpack-dev-server-fresh').devServer, objApi: true }
+        return { devServer: require('@cypress/webpack-dev-server').devServer, objApi: true }
       }
 
       if (devServer.bundler === 'vite') {
-        return { devServer: require('@cypress/vite-dev-server-fresh').devServer, objApi: true }
+        return { devServer: require('../../../../../npm/vite-dev-server').devServer, objApi: true }
```

Comment:
Should this be relative?

Reply:
Addressed in 2b8faab2eb3136494e36441f0ad5518a8a2c83b5